django_select2/static/django_select2/django_select2.js

```diff
@@ -51,6 +63,7 @@
   }
 
   $(function () {
-    $('.django-select2').djangoSelect2()
+    // do not trigger select2 initialization in hidden admin inline formset rows
+    $('.django-select2').not('.empty-form .django-select2').djangoSelect2()
```

Comment:
Why so? Is there a side effect? This is missing in the version I am writing for Django 2. It would be good to know.